exercises/F-logical-operators/exercise.js

```diff
@@ -11,14 +11,14 @@ var cssLevel = 4;
 
 // Finish the statement to check whether HTML, CSS knowledge are above 5
 // (hint: use the comparison operator from before)
-var htmlLevelAbove5;
-var cssLevelAbove5;
+var htmlLevelAbove5 = true;
+var cssLevelAbove5 = false;
 
 // Finish the next two statement
 // Use the previous variables and logical operators
 // Do not "hardcode" the answers
-var cssAndHtmlAbove5;
-var cssOrHtmlAbove5;
+var cssAndHtmlAbove5 = htmlLevel > 5 && cssLevel > 5 ? true : false;
```

Comment:
`var cssAndHtmlAbove5 = htmlLevelAbove5 && cssLevelAbove5 ` gives you the same result. the ternary operator you've used simplifies it but if you were to use the usual if statements it would lead to longer lines of code.
same applies to the next expression

Reply:
Thanks bro I catch your drift